cfpb/cfgov-refresh
# Add optional 'eyebrow' to Text Introduction module

While migrating the BAH journey content into Wagtail, we opted to use an H5 eyebrow above the H1 header in the Text Introduction module to give users additional information about the current step. We initially entered the eyebrow and heading content in the body field. This PR adds an optional `pre-heading/eyebrow` field to Text Introduction module to standardize the markup and make the eyebrow pattern available elsewhere in the site.

## Additions

- Eyebrow option in Text Introduction module



## Testing

1. Check out branch and run migrations
2. Add a page with a TextIntroduction and verify that eyebrow displays correctly
3. Verify that tests pass

## Screenshots

### Admin
<img width="1054" alt="screen shot 2018-04-12 at 8 40 42 am" src="https://user-images.githubusercontent.com/778171/38689103-8975df06-3e2f-11e8-8766-71d346d8ec1f.png">

### Desktop
<img width="731" alt="screen shot 2018-04-12 at 8 43 11 am" src="https://user-images.githubusercontent.com/778171/38689080-7c73f9aa-3e2f-11e8-99cc-c3479b252cab.png">

### Mobile
<img width="386" alt="screen shot 2018-04-12 at 8 44 29 am" src="https://user-images.githubusercontent.com/778171/38689083-7f05c6da-3e2f-11e8-8b11-a786323d6911.png">


## Checklist


## Testing checklist

### Browsers


### Accessibility


### Other
